### PR TITLE
Fix/5251 wrong text risk detail screen with low encounter

### PR DESCRIPTION
**Pull request by tcfos**
## Description
Low risk with 1 or more encounters. Issues:

1. Risk details card shows wrong text for section "Infektionsrisiko"
2. Risk details card shows a link to the FAQ.

Fixed: 

1. fixed the text
2. removed the FAQ link

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-5251

## Screenshots

Simulator Screen Shot - iPhone 12 - 2021-02-22 at 16 05 24

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/__tests__/ExposureDetectionViewModelTests.swift [hunk 1]
```diff
@@ -898,20 +898,20 @@ class ExposureDetectionViewModelTests: XCTestCase {
...
+//		if hasAtLeastOneDayWithLowRiskLevel {
+//			XCTAssertEqual(section.cells[2].cellReuseIdentifier.rawValue, "linkCell")
+//
+//			switch section.cells[2].action {
+//			case .open(url: let url):
+//				XCTAssertEqual(AppStrings.ExposureDetection.explanationFAQLink, url?.absoluteString)
+//			default:
+//				XCTFail("FAQ Link cell not found")
+//			}
+//		}
```
Comment (KaiTeuber) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/__tests__/ExposureDetectionViewModelTests.swift:914]: Do we still need this?

**Review by KaiTeuber — DISMISSED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/__tests__/ExposureDetectionViewModelTests.swift (hunk 1 above)
Comment (tcfos) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/__tests__/ExposureDetectionViewModelTests.swift:914]: of course not. Thought I had it removed already...

Comment (tcfos) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/__tests__/ExposureDetectionViewModelTests.swift]: of course not. Thought I has it removed already...

**Review by janetback — APPROVED** [review]
UA approved

**Review by naveeddotio — APPROVED** [review]

**Pull request merged**